lib/java/jvm-channel/src/main/java/org/enso/jvm/channel/JVM.java

```diff
@@ -30,10 +30,19 @@ public final class JVM {
    * @return new instance of the JVM
    */
   public static JVM create(File javaHome, String... options) {
```

Comment:
So if `javaHome` is no0t a directory, it is not considered a JVM but rather a "library with embedded SVM"? Please update the docs, the naming is very confusing now.